test/frames.jl

```diff
@@ -0,0 +1,74 @@
+@testset "Check RFrame shorthands" begin
+    function dummy(args...) end
+    video = Video(10, 10)
+    Background(1:20, dummy)
+    a = Object(1:3, (args...) -> O) # 1:3
+    b = Object(RFrames(1:3), (args...) -> O) # 4:6
+    c = Object(RFrames(1:3, prev_start()), (args...) -> O) # 5:7
+    d = Object(RFrames(prev_start(), prev_last()), (args...) -> O) # 5:7
+    render(video; tempdirectory = "images", pathname = "")
+    @test a.frames.frames == 1:3
+    @test b.frames.frames == 4:6
+    @test c.frames.frames == 5:7
+    @test d.frames.frames == 5:7
+end
+
+@testset "Check RFrame helper function for previous object" begin
+    function dummy(args...) end
+    video = Video(10, 10)
+    Background(1:20, dummy)
+    a = Object(1:3, (args...) -> O) # 1:3
+    b = Object(RFrames(2:5, prev_start(), default_last()), (args...) -> O) # 3:6
```

Comment:
I find this a little hard to read. What exactly is the meaning of `default_last()` when combined with `prev_start` here?